model_analyzer/result/result_manager.py

```diff
@@ -336,9 +340,13 @@ def _setup_for_sequential_analysis(self):
             model.model_name() for model in self._config.analysis_models
         ]
 
+        # If analyzing sequentially just set all models to equal weighting (since it won't be used)
```

Comment:
Any reason this code can't be the same as the concurrent code now that we have defaults?  Even if they specify some other value like 2, it won't change anything.

Reply:
That's true. I've removed this code.